src/main/java/domain/Letter.java

```diff
@@ -0,0 +1,45 @@
+package domain;
+
+import java.util.Arrays;
+import java.util.List;
+
+public enum Letter {
+    TWO("2", 2),
+    THREE("3", 3),
+    FOUR("4", 4),
+    FIVE("5", 5),
+    SIX("6", 6),
+    SEVEN("7", 7),
+    EIGHT("8", 8),
+    NINE("9", 9),
+    TEN("10", 10),
+    J("J", 10),
+    Q("Q", 10),
+    K("K", 10),
+    A("A", 11);
```

Comment:
```suggestion
    JACK("J", 10),
    QUEEN("Q", 10),
    KING("K", 10),
    ACE("A", 11);
```

위 enum 객체 값들과 통일성을 이루기 위해서 이렇게 작성해보는 것은 어떨까요?

Reply:
저 또한 코드를 구현할 때, 구조적인 측면이나 이런 디테일한 측면 모두 통일성, 일관성을 갖도록 하는 것을 선호합니다.!

말씀해주신 부분 수정하도록 했고, 추가적으로 찾아보니깐 저희가 트럼프 카드의 도메인 용어를 놓쳤던 부분이 있더라고요!?

기존 용어 -> 변경된 용어
Letter -> Rank
Mark -> Suit
CLOVER -> CLUBS
HEART -> HEARTS
DIAMOND -> DIAMONDS
SPAED -> SPADES
J -> JACK
K -> KING
Q -> QUEEN
A -> ACE

이런식으로 바꾸고 최대한 도메인 용어를 활용할 수 있도록 변경했습니다.